### PR TITLE
Improved pvp combat logging

### LINKED ISSUE #7: Improved combat logging
What do you think about CSV combat logs? Considering the sheer mass of log entries which will be collected, it could be better to save the logs in the CSV format, which means that server admins can open the file with a tool like Excel and sort/filter the informations contained in it like they want. Also it is better to  either save the full data or - better - rotate the log files each day, which means that every day a new file will be created with the current date contained in the file name. Of course this would be configurable.
So, what do you think about it? If you want, I'll gladly provide a PR for this (unless you want to work on this by your own).

### PR DESCRIPTION
**Pull request by CraftedMods**
I've improved the current pvp logging API, added support for multiple handlers and added CSV handler. Closes #7.
~~@VulcanForge **Important:** Please merge the other PRs from me before this one! #17  introduced changes which are not compatible with this branch, resulting in merge conflicts. I'll resolve them if you've merged the other PRs.~~

### CODE REVIEW

File: main/java/pvpmode/log/AbstractFileCombatLogHandler.java [hunk 1]
```diff
@@ -66,9 +66,12 @@ public void init(Path pvpLoggingDir)
     protected Path getUnusedFileNameWithIndex(Path basicFileNameWithoutFileEnding, String fileEnding)
     {
         Path result = basicFileNameWithoutFileEnding;
-        for (int i = 1; Files.exists (result = Paths
-                        .get (basicFileNameWithoutFileEnding.toString () + "_" + i + "." + fileEnding)); i++)
-            ;
+        int i = 1;
+        do
+        {
+            result = Paths.get (basicFileNameWithoutFileEnding.toString () + "_" + i++ + "." + fileEnding);
```
Comment (CraftedMods) [inline main/java/pvpmode/log/AbstractFileCombatLogHandler.java]: Also, the `+i++ +` looks funny.

Comment (Mahtaran) [reply main/java/pvpmode/log/AbstractFileCombatLogHandler.java]: I'd use String.format() for these types of sentences (everything with more than 2 variables)

Comment (CraftedMods) [reply main/java/pvpmode/log/AbstractFileCombatLogHandler.java]: This is what I did with the next commit.

**Comment by Mahtaran** [comment]
Oh lol

**Pull request merged**